Mapping/SolARPipeline_Mapping_Multi/src/PipelineMappingMultiProcessing.cpp

```diff
@@ -390,10 +404,10 @@ int m_nbImageRequest(0), m_nbExtractionProcess(0), m_nbFrameToUpdate(0),
     }
 
     FrameworkReturnCode PipelineMappingMultiProcessing::mappingProcessRequest(const std::vector<SRef<SolAR::datastructure::Image>> & images,
-                                                                              const std::vector<SolAR::datastructure::Transform3Df> & poses,
+                                                                              const std::vector<SolAR::datastructure::Transform3Df> & poses_ARr,
```

Comment:
Currently the convention for member names is camel-case, not snake-case. But I understand this might look better that way.
This would look like:
`aRrPoses/posesArr`, `aRrToWorldTransform/transformARrToWorld`, `aRrToWorldUpdateTransform/updatedTransformARrToWorld` ...
I let you decide to keep snake-case or pick on camel-case form (or find better ones)